samples/client/others/typescript/builds/array-of-lists/DefaultApi.md

```diff
@@ -15,17 +15,15 @@ Method | HTTP request | Description
 
 
 ```typescript
-import {  } from '';
```

Comment:
most of these samples do not define a `moduleName` or `projectName` in their yaml. Examples are also sparse. The petstore is pretty decent.